### ISSUE TITLE
PHP Fatal error:  Uncaught TypeError: htmlspecialchars() expects parameter 1 to be string, null given

### ISSUE THREAD

**Issue by Cugar15** [phpmyadmin/phpmyadmin#17118]
### Describe the bug

There is one table in the database which triggers a fatal error when trying to display, search.

```
2021/09/20 12:35:44 [error] 2298#2298: *2176 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught TypeError: htmlspecialchars() expects parameter 1 to be string, null given in /usr/share/phpMyAdmin-5.1.1/libraries/classes/Display/Results.php:2012
Stack trace:
#0 /usr/share/phpMyAdmin-5.1.1/libraries/classes/Display/Results.php(2012): htmlspecialchars()
#1 /usr/share/phpMyAdmin-5.1.1/libraries/classes/Display/Results.php(1682): PhpMyAdmin\Display\Results->getSortOrderLink()
#2 /usr/share/phpMyAdmin-5.1.1/libraries/classes/Display/Results.php(1058): PhpMyAdmin\Display\Results->getOrderLinkAndSortedHeaderHtml()
#3 /usr/share/phpMyAdmin-5.1.1/libraries/classes/Display/Results.php(1174): PhpMyAdmin\Display\Results->getTableHeadersForColumns()
#4 /usr/share/phpMyAdmin-5.1.1/libraries/classes/Display/Results.php(4211): PhpMyAdmin\Display\Results->getTableHeaders()
#5 /usr/share/phpMyAdmin-5.1.1/libraries/classes/Sql.php(1364): PhpMyAdmin\Display\Results->getTable()
#6 /usr/share/phpMyAdmin-5.1.1/libraries/classes/Sql.php(1620): PhpMyAdmin\Sql->getHtmlForSqlQuer" while reading response header from upstream, client: 188.999.99.99, server: www.xxxxx.de, request: "GET /xxxx/index.php?route=/sql&server=1&db=www_xxx&table=table_items&pos=0&ajax_request=true&ajax_page_request=true&_nocache=1632134143959825635&token=74624f47437f5f484a306f42796a3637 HTTP/1.1", upstream: "fastcgi://unix:/run/php/php7.4-fpm.sock:", host: "www.xxx.de"

``` 

This happens with phpmyadmin version 5.0.4 and after ugrade to 5.1.1 same way.
Seems to be that only one table is affected.
Clearing cache, cookies, etc did not help.

- Web server: nginx/1.12.1
- Database version:  5.7.30-log - MySQL Community Server (GPL)
- PHP version: 7.3 and 7.4
- phpMyAdmin version: 5.0.3 and 5.1.1

Thanks for helping!

**Comment by mauriciofauth** [comment]
Duplicate of #17046

**Comment by mauriciofauth** [comment]
Thank you for reporting this issue, @Cugar15. This will be fixed in the next release (phpMyAdmin 5.1.2).
